src/editor.html

```diff
@@ -1696,6 +1710,10 @@
 				padding-left: 10px;
 				padding-right: 10px;
 			}
+
+			#zss_editor_footer {
+				height: 10px;
```

Comment:
why do we need a hard-coded value for footer, and why even handle the footer if we don't use it?
worried about messing with layouts in other screens that are using this component